src/pages/signin/ResendValidationForm.js

```diff
@@ -46,92 +47,56 @@ const defaultProps = {
+    const loginType = (isSMSLogin ? props.translate('common.phone') : props.translate('common.email')).toLowerCase();
+    const error = ErrorUtils.getLatestErrorMessage(props.account);
+    const successMessage = props.account.message;
+
+    return (
+        <>
+            <View style={[styles.mt3, styles.flexRow, styles.alignItemsCenter, styles.justifyContentStart]}>
+                <Avatar
+                    source={ReportUtils.getDefaultAvatar(props.credentials.login)}
+                    imageStyles={[styles.mr2]}
+                />
+                <View style={[styles.flex1]}>
+                    <Text style={[styles.textStrong]}>
+                        {login}
                     </Text>
                 </View>
-                {!_.isEmpty(this.state.formSuccess) && (
-                    <Text style={[styles.formSuccess]}>
-                        {this.state.formSuccess}
+            </View>
+            <View style={[styles.mv5]}>
+                <Text>
+                    {props.translate('resendValidationForm.weSentYouMagicSignInLink', {login, loginType})}
+                </Text>
+            </View>
+            {successMessage && (
+                <DotIndicatorMessage style={[styles.mb5]} type="success" messages={[successMessage]} />
+            )}
+            {!successMessage && error && (
+                <DotIndicatorMessage style={[styles.mb5]} type="error" messages={[error]} />
+            )}
+            <View style={[styles.mb4, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter]}>
+                <TouchableOpacity onPress={() => redirectToSignIn()}>
+                    <Text>
+                        {props.translate('common.back')}
                     </Text>
-                )}
-                <View style={[styles.mb4, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter]}>
-                    <TouchableOpacity onPress={() => redirectToSignIn()}>
-                        <Text>
-                            {this.props.translate('common.back')}
-                        </Text>
-                    </TouchableOpacity>
-                    <Button
-                        medium
-                        success
-                        text={this.props.translate('resendValidationForm.resendLink')}
-                        isLoading={this.props.account.loading}
-                        onPress={this.validateAndSubmitForm}
-                        isDisabled={this.props.network.isOffline}
-                    />
-                </View>
-                <OfflineIndicator containerStyles={[styles.mv1]} />
-            </>
-        );
-    }
-}
+                </TouchableOpacity>
+                <Button
+                    medium
+                    success
+                    text={props.translate('resendValidationForm.resendLink')}
+                    isLoading={props.account.loading}
```

Comment:
```suggestion
                    isLoading={props.account.isLoading}
```
You are setting `isLoading` in Onyx, not `loading`

Reply:
Ah this was already there before my changes.. but good catch regardless!